### DISCUSSION TITLE
Blue Ray with ISO file inside

### DISCUSSION THREAD

**Discussion by xraves** [aldostools/webMAN-MOD#752] [Q&A]
Can I recorder a Blu Ray with several ISO files inside and use on webMAN-MOD ?
How I do it?

**Comment by aldostools** [comment]
No. If you mount an ISO stored on a CD, DVD or BD, the source file is lost because the ISO cannot be read from /dev_bdvd.  

The ISO can be mounted from sd, ms, cf, fat32, ntfs, exfat, ext2, ext3, ext4, internal HDD or remote ps3netsrv.

Maybe if the games are extracted packages you could redirect /dev_bdvd/GAMEI to /dev_hdd0/game or mount the game to launch it from /app_home. This option could work with limations due the disc is read only and some games requrie the game folder to be read/write.
